src/main/java/com/marginallyclever/communications/tcp/TCPTransportLayer.java

```diff
@@ -49,7 +50,7 @@ public NetworkSession openConnection(String connectionName) {
 	}
 
 	@Override
-	public TransportLayerPanel getTransportLayerPanel() {
-		return new TCPTransportLayerPanel(this);
+	public List<String> listConnections() {
+		return List.of("192.168.1.183:9999");
```

Comment:
what is this hard coded ip?

Reply:
It's taken from here: https://github.com/MarginallyClever/Makelangelo-software/blob/c391867f69137d7d5e606ce262314cb7a8683fc8/src/main/java/com/marginallyclever/communications/tcp/TCPTransportLayerPanel.java#L20